src/contention_detectors/overload.hpp

```diff
@@ -48,23 +32,34 @@ class OverloadDetector :
   OverloadDetector(
       Consumer<Contentions>* _consumer,
       const lambda::function<usage::GetterFunction>& _cpuUsageGetFunction,
-      SerenityConfig _conf,
+      const SerenityConfig& _conf,
       const Tag& _tag = Tag(QOS_CONTROLLER, NAME))
     : tag(_tag),
       cpuUsageGetFunction(_cpuUsageGetFunction),
       Producer<Contentions>(_consumer) {
-    SerenityConfig config = OverloadDetectorConfig(_conf);
-    this->cfgUtilizationThreshold =
-      config.getD(detector::THRESHOLD);
+    configure(_conf);
   }
 
   ~OverloadDetector() {}
 
-  Try<Nothing> consume(const ResourceUsage& in) override;
-
   static const constexpr char* NAME = "OverloadDetector";
 
  protected:
+  void configure(const SerenityConfig& externalConf) {
+    SerenityConfig config = SerenityConfig();
+
+    //! double_t
+    //! Detector threshold.
+    config.set(detector::THRESHOLD, detector::DEFAULT_UTILIZATION_THRESHOLD);
```

Comment:
Preparation of default SerenityConfig should be in separate function.
Also, I'm not sure if we want to go this way :)